README.md

```diff
@@ -12,73 +10,12 @@ This plugin has the following minimum requirements:
 - v3.0.0+: Grafana 7.1.0
 - < v3.0.0 require Grafana 6.3.6.
 
-### Grafana Cloud
-
-If you do not have a [Grafana Cloud](https://grafana.com/cloud) account, you can sign up for one [here](https://grafana.com/cloud/grafana).
-
-1. Click on the `Install plugin` button on the [Azure Data Explorer page on Grafana.com](https://grafana.com/plugins/grafana-azure-data-explorer-datasource/?tab=installation). This will automatically add the plugin to your Grafana instance. It might take up to 30 seconds to install.
-
-2. Login to your Hosted Grafana instance (go to your instances page in your profile): `https://grafana.com/orgs/<yourUserName>/instances/` and the Azure Data Explorer datasource will be installed.
-
-### Installation Instructions on the Grafana Docs Site
-
-- [Installing on Debian/Ubuntu](http://docs.grafana.org/installation/debian/)
-- [Installing on RPM-based Linux (CentOS, Fedora, OpenSuse, RedHat)](http://docs.grafana.org/installation/rpm/)
-- [Installing on Windows](http://docs.grafana.org/installation/windows/)
-- [Installing on Mac](http://docs.grafana.org/installation/mac/)
-
-### Docker
-
-1. Fetch the latest version of grafana from Docker Hub:
-   `docker pull grafana/grafana:latest`
-2. Run Grafana and install the Azure Data Explorer plugin with this command:
-
-   ```bash
-   docker run -d --name=grafana -p 3000:3000 -e "GF_INSTALL_PLUGINS=grafana-azure-data-explorer-datasource" grafana/grafana:latest
-   ```
-
-3. Open the browser at: <http://localhost:3000> or <http://your-domain-name:3000>
-4. Login in with username: `admin` and password: `admin`
-5. To make sure the plugin was installed, check the list of installed datasources. Click the Plugins item in the main menu. Both core datasources and installed datasources will appear.
-
-This ia an alternative command if you want to run Grafana on a different port than the default 3000 port:
-
-```bash
-docker run -d --name=grafana -p 8081:8081 -e "GF_SERVER_HTTP_PORT=8081" -e "GF_INSTALL_PLUGINS=grafana-azure-data-explorer-datasource" grafana/grafana:master
-```
-
-It is recommended that you use a volume to save the Grafana data in. Otherwise if you remove the docker container, you will lose all your Grafana data (dashboards, users etc.). You can create a volume with the [Docker Volume Driver for Azure File Storage](https://github.com/Azure/azurefile-dockervolumedriver).
-
-#### Docker latest
-
-Alternatively, you can use the tag `azure-data-explorer-datasource-latest` to pull down a Docker image for whatever is current in `main`.
-
-`docker run --rm -p 3000:3000 grafana/plugin-builds:azure-data-explorer-datasource-latest`
```

Comment:
I don't think the grafana docs touch on Docker, do we want to move this info to a different file called `DOCKER.md`?

Reply:
That information is [here](https://grafana.com/docs/grafana/latest/setup-grafana/installation/docker/#install-plugins-in-the-docker-container) so I think it's fine to remove. The only difference I'd say is the ability to pull down whatever's in current main for ADX, that's something I don't think is generally available for all plugins?

Reply:
Ah I knew I was missing something Docker related somewhere, thank you! Yes, the only difference is we have a "pointer container"  to `main`. Maybe we leave that in? Or should we move it out?

Reply:
I think we should move this out. I don't think someone trying to set up Grafana with the ADX plugin will likely find this information. Also, would a user ever want to use the main instead of the latest release? I feel like this is a note for developers.